cli/src/api/templates/js-binding.ts

```diff
@@ -238,7 +238,7 @@ if (!nativeBinding || process.env.NAPI_RS_FORCE_WASI) {
       wasiBindingError = err
     }
   }
-  if (!nativeBinding) {
+  if (!nativeBinding || process.env.NAPI_RS_FORCE_WASI) {
```

Comment:
Otherwise it will not try to load `wasm` from installed package if there is already found native package